src/journeys_utils.js

```diff
@@ -266,12 +266,44 @@ journeys_utils.addHtmlToIframe = function(iframe, html, userAgent) {
 	else {
 		bodyClass = 'branch-banner-desktop';
 	}
-
+	var instacartAccessibilityTemplateId = '1042917133147779073';
```

Comment:
1. Let's remove instacart's name from the variable.
2. We should also make it an array if we need to quickly add other templates.